packages/web/src/pages/audio-rewards-page/components/modals/CognitoModal.tsx

```diff
@@ -62,17 +63,21 @@ export const CognitoModal = () => {
       switch (event.action) {
         case 'opened':
           dispatch(setCognitoFlowStatus({ status: CognitoFlowStatus.OPENED }))
-          track(Name.REWARDS_CLAIM_START_COGNITO_FLOW, {
-            handle,
-            source
-          })
+          record(
+            make(Name.REWARDS_CLAIM_START_COGNITO_FLOW, {
+              handle,
+              source
+            })
+          )
           break
         case 'closed':
           dispatch(setCognitoFlowStatus({ status: CognitoFlowStatus.CLOSED }))
-          track(Name.REWARDS_CLAIM_FINISH_COGNITO_FLOW, {
-            handle,
-            source
-          })
+          record(
+            make(Name.REWARDS_CLAIM_FINISH_COGNITO_FLOW, {
```

Comment:
just wondering if make should just be a part of record at some point?

Reply:
Definitely think so, can address as part of the other issue